src/app/lib/config/services/hausa.js

```diff
@@ -41,29 +41,27 @@ const service = {
     error: {
       404: {
         statusCode: '404',
-        title: '404 - Ba a samu shafin ba',
+        title: '404 - Ba za a samu wannan shafin ba',
         message:
-          'Mai yiwuwa ba ku rubuta adireshin daidai ba. Sake duba adireshin da kuma rubutun.',
+          'Afuwa, ba za mu iya kawo maku wannan shafin da kuke nema ba. Sake gwadawa:',
         solutions: [
-          'Double checking the url',
-          'Hitting the refresh button in your browser',
-          'Searching for this page using the BBC search bar',
+          'Sake duba adireshin',
+          'Sabunta shafin',
+          'Bincika shafin ta hanyar amfani da gurbin binciken BBC',
         ],
-        callToActionFirst: 'Alternatively, please visit the ',
-        callToActionLinkText: 'Labaran Duniya',
+        callToActionFirst: 'Maimakon haka, ziyarci shafin sashen',
```

Comment:
Hey! Seems like there should be a space in the end of the callToActionFirst variable ('Maimakon haka, ziyarci shafin sashen ') since it will be followed by a link on the webpage. 
Other than that everything looks good:)

Reply:
@OlgaLyubin Thanks. I've made the change now.